### PR TITLE
API documentation fixes.

### PR DESCRIPTION
**Pull request by pcolast**

### CODE REVIEW

File: modules/ROOT/attachments/specs/cvs-ted-europa-eu.yaml [hunk 1]
```diff
@@ -1,20 +1,22 @@
...
 servers:
   - url: https://cvs.ted.europa.eu
 paths:
   /v1/notices/validation:
     post:
       tags:
-        - notice-rest-controller
-      summary: Validate the notice
-      description: Validate a notice sent on body. The request returns a schematron validation report which describes all the tests performed.
+        - notice-validate-controller
```
Comment (bertrand-lorentz) [inline modules/ROOT/attachments/specs/cvs-ted-europa-eu.yaml]: There is no need to have "-controller" in the tags. Tags are just a way to group operation.
So I'd suggest to put either "Notice", or "Validation"

File: modules/ROOT/attachments/specs/enotices2-ted-europa-eu-esenders.yaml [hunk 2]
```diff
@@ -12,47 +12,87 @@ servers:
...
+            - Several versions of notice with given NoticeId exist.
+          content:
+            application/json: {}
+         
+  /api/notice/notices/{NoticeId}/stop-publication-esentool:
     post:
       tags:
         - Notice
-      summary: send stop notice publication to ted monitor
+      summary: Request stop notice publication
```
Comment (bertrand-lorentz) [inline modules/ROOT/attachments/specs/enotices2-ted-europa-eu-esenders.yaml]: This is a bit unclear, I think a better phrasing would be "Request to stop the publication of a notice"

File: modules/ROOT/attachments/specs/enotices2-ted-europa-eu-esenders.yaml [hunk 3]
```diff
@@ -12,47 +12,87 @@ servers:
 security:
   - esenders-token: []
 paths:
-  /api/notice/notices/{businessId}/validation-report:
+  /api/notice/notices/{NoticeId}/validation-report:
```
Comment (bertrand-lorentz) [inline modules/ROOT/attachments/specs/enotices2-ted-europa-eu-esenders.yaml]: Although it doesn't make a difference here, I'd suggest to always start parameter names with lowercase, so "noticeId".
This is consistent with other parameter names : "sortProperty", etc.
Same for other changes to "NoticeId" below.

File: modules/ROOT/attachments/specs/enotices2-ted-europa-eu-esenders.yaml [hunk 4]
```diff
@@ -101,6 +158,7 @@ paths:
             format: int32
             default: 0
         - name: size
+          description: Number of search results per page.
```
Comment (bertrand-lorentz) [inline modules/ROOT/attachments/specs/enotices2-ted-europa-eu-esenders.yaml]: To be precise, this should rather say: "Maximum number of search results per page"

File: modules/ROOT/attachments/specs/viewer-ted-europa-eu.yaml [hunk 5]
```diff
@@ -19,10 +19,11 @@ paths:
     post:
       tags:
         - notice-render-controller
-      summary: Request to render a notice in HTML or PDF
+      summary: Get HTML or PDF rendering of a notice sent on body.
```
Comment (bertrand-lorentz) [inline modules/ROOT/attachments/specs/viewer-ted-europa-eu.yaml]: It should be "... sent in the request body."

File: modules/ROOT/pages/index.adoc [hunk 6]
```diff
@@ -2,20 +2,20 @@
...
- * Validate, render, or translate notices.
+ * Validate or render notices.
 
 TED API allows anonymous access to all services manipulating published notices (i.e. searching for, or retrieving notices already published on TED). These APIs are mostly targeted to data reusers. You can find information on reusing TED data at the link:https://simap.ted.europa.eu/en_GB/web/simap/developers-corner#get-published[SIMAP Developers' Corner].
 
 All TED APIs manipulating notices that are not yet published, require the \cas\logincaller to be authenticated with an API Key.
 
 
-## API Keys
+== API Keys
```
Comment (bertrand-lorentz) [inline modules/ROOT/pages/index.adoc]: The case of "Key" is inconsistent in the titles: "Keys" here, "KEY" in the title below, and "key" in the title after that.
I'd suggest the normal all lowercase "key"

File: modules/ROOT/attachments/specs/enotices2-ted-europa-eu-esenders.yaml [hunk 7]
```diff
@@ -109,11 +167,13 @@ paths:
             format: int32
             default: 20
         - name: sortProperty
+          description: Selected property used in ordering the search results.
```
Comment (bertrand-lorentz) [inline modules/ROOT/attachments/specs/enotices2-ted-europa-eu-esenders.yaml]: I'd suggest "Property to use to sort the search results."
If we know, we should describe what are the possible values, and what the behavior is when this parameter is not indicated.

**Review by bertrand-lorentz — CHANGES_REQUESTED** [review]

File: modules/ROOT/attachments/specs/cvs-ted-europa-eu.yaml (hunk 1 above)
Comment (pcolast) [reply modules/ROOT/attachments/specs/cvs-ted-europa-eu.yaml]: OK

**Review by bertrand-lorentz — APPROVED** [review]

File: modules/ROOT/attachments/specs/enotices2-ted-europa-eu-esenders.yaml [hunk 8]
```diff
@@ -12,47 +12,87 @@ servers:
...
+          description: Notice identifier
           in: path
           required: true
           schema:
             type: string
             format: uuid
       responses:
         '200':
-          description: OK
+          description: Notice publication stopped
```
Comment (meletev) [inline modules/ROOT/attachments/specs/enotices2-ted-europa-eu-esenders.yaml]: Please change to "Notice publication stopped successfully", to make it clear that the request was successful.

File: modules/ROOT/attachments/specs/enotices2-ted-europa-eu-esenders.yaml [hunk 9]
```diff
@@ -12,47 +12,87 @@ servers:
 security:
   - esenders-token: []
 paths:
-  /api/notice/notices/{businessId}/validation-report:
+  /api/notice/notices/{noticeId}/validation-report:
```
Comment (meletev) [inline modules/ROOT/attachments/specs/enotices2-ted-europa-eu-esenders.yaml:15]: The current specification uses "businessId", not "noticeId".

File: modules/ROOT/attachments/specs/enotices2-ted-europa-eu-esenders.yaml [hunk 10]
```diff
@@ -12,47 +12,87 @@ servers:
...
     get:
       tags:
         - Notice
       summary: Get notice validation report
+      description: Get validation report created during notice submission request
       operationId: getValidationReport
       parameters:
-        - name: businessId
+        - name: noticeId
```
Comment (meletev) [inline modules/ROOT/attachments/specs/enotices2-ted-europa-eu-esenders.yaml:23]: The current specification uses "businessId", not "noticeId".

File: modules/ROOT/attachments/specs/enotices2-ted-europa-eu-esenders.yaml [hunk 11]
```diff
@@ -77,14 +117,30 @@ paths:
                   type: string
                   format: binary
                 metadata:
-                  $ref: '#/components/schemas/EsenderAuthor'
+                  $ref: '#/components/schemas/NoticeAuthor'
```
Comment (meletev) [inline modules/ROOT/attachments/specs/enotices2-ted-europa-eu-esenders.yaml]: The current specification defines "EsenderAuthor", not "NoticeAuthor".

File: modules/ROOT/attachments/specs/enotices2-ted-europa-eu-esenders.yaml [hunk 12]
```diff
@@ -179,7 +246,7 @@ components:
           type: array
           items:
             type: string
-    EsenderAuthor:
+    NoticeAuthor:
```
Comment (meletev) [inline modules/ROOT/attachments/specs/enotices2-ted-europa-eu-esenders.yaml]: The current specification defines "EsenderAuthor", not "NoticeAuthor".

**Review by meletev — CHANGES_REQUESTED** [review]

File: modules/ROOT/attachments/specs/enotices2-ted-europa-eu-esenders.yaml (hunk 12 above)
Comment (rousso) [reply modules/ROOT/attachments/specs/enotices2-ted-europa-eu-esenders.yaml]: I think Vangelis is right for this one. You are changing a schema. You can't

File: modules/ROOT/attachments/specs/enotices2-ted-europa-eu-esenders.yaml (hunk 11 above)
Comment (rousso) [reply modules/ROOT/attachments/specs/enotices2-ted-europa-eu-esenders.yaml]: I think Vangelis is right for this one too. you cannot change the request body of the current API.
We only want to improve the documentation part (only titles, summaries, descriptions etc).

File: modules/ROOT/attachments/specs/enotices2-ted-europa-eu-esenders.yaml (hunk 9 above)
Comment (rousso) [reply modules/ROOT/attachments/specs/enotices2-ted-europa-eu-esenders.yaml:15]: I think that renaming this parameter @meletev  does not have any impact as it os only a placeholder in the URL. I think we can keep the change proposed by @pcolast

File: modules/ROOT/attachments/specs/enotices2-ted-europa-eu-esenders.yaml (hunk 10 above)
Comment (rousso) [reply modules/ROOT/attachments/specs/enotices2-ted-europa-eu-esenders.yaml:23]: I think that renaming this parameter @meletev  does not have any impact as it os only a placeholder in the URL. I think we can keep the change proposed by @pcolast

File: modules/ROOT/attachments/specs/enotices2-ted-europa-eu-esenders.yaml (hunk 8 above)
Comment (rousso) [reply modules/ROOT/attachments/specs/enotices2-ted-europa-eu-esenders.yaml]: I agree with @meletev here. Description of status 200 should always be clear that the request was successful. Changing it rom "OK" to "blah blah stopped" is confusing.

**Review by rousso — CHANGES_REQUESTED** [review]

**Review by meletev — APPROVED** [review]

**Review by rousso — APPROVED** [review]

**Pull request merged**